sakai-evaluation-impl/src/java/org/sakaiproject/evaluation/logic/EvalDeliveryServiceImpl.java

```diff
@@ -572,6 +572,15 @@ protected boolean checkAnswersValidForEval(EvalResponse response, boolean checkR
                 throw new IllegalArgumentException("NULL templateItem or templateItem.item for answer: " 
                         + "Answers must have the templateItem set, and that must have the item set");
             }
+            
```

Comment:
This code change (in EvalDeliveryServiceImpl) doesn't appear to belong to this issue so shouldn't be part of this PR.